src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java

```diff
@@ -90,4 +97,21 @@ public static void clearDocumentTransientFields(SscsCaseData caseData) {
         caseData.setDocumentGeneration(DocumentGeneration.builder().build());
         caseData.setDocumentStaging(DocumentStaging.builder().build());
     }
+
+    public static void excludePanelMembers(PanelMemberExclusions exclusions, List<JudicialUserBase> panelMembers) {
+        if (nonNull(panelMembers)) {
```

Comment:
What happens if panelMembers is null?

Reply:
We want to keep the list the same but set the exclusions to yes